media/css/exp/home/home-2018.scss

```diff
@@ -37,281 +20,26 @@ $image-path: '/media/protocol/img';
     margin-top: $spacing-lg;
 }
 
-
-//* -------------------------------------------------------------------------- */
-// Primary CTA (page top)
-
-.c-primary-cta {
```

Comment:
Little too enthusiastic 😅  The primary-cta stuff is still used by non-Firefox browsers.